pom.xml

```diff
@@ -119,6 +149,11 @@
 			<version>1.49</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>com.google.code.gson</groupId>
+			<artifactId>gson</artifactId>
+			<version>2.8.7</version>
+		</dependency>
```

Comment:
same here-Please check which gson version we were using before this update

Reply:
We are currently using 2.8.5 and I will update the pom.xml with the same.
Gson dependency had to be explicitly added now since in grpc v1.20.0 gson was a compile time dependency where as it no longer is in v1.40.0